### PR TITLE
Configure universal wheel building

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
```
Comment (mattrobenolt) [inline setup.cfg]: Hmm, why is this `[bdist_wheel]` now? I used to add just `[wheel]` to projects.

/cc @dstufft

Comment (graingert) [inline setup.cfg]: me too, but bdist_wheel is the documented option https://bitbucket.org/pypa/wheel/src/f746a01eab7a673cb628a52f99c129d0d5741bcf/setup.cfg?at=default#cl-7

Comment (mattrobenolt) [inline setup.cfg]: Yeah, it's also documented on http://pythonwheels.com/.

Just curious why this changed. :)

**Comment by graingert** [comment]
:cake: :sparkles:

File: setup.cfg (hunk 1 above)
Comment (dstufft) [inline setup.cfg]: The "standard" name of the section is to name them after the command name (e.g. you can have an ``[sdist]`` or an ``[upload]`` section or whatever). The old name was just something Daniel did and it got switched to be more inline with what the rest of the commands used.

Comment (graingert) [inline setup.cfg]: ah neat, thanks @dstufft

**Pull request merged**